README.md

```diff
@@ -55,3 +55,10 @@ Build DropDownMenuKit framework and drop it into your Xcode project.
 #### Files
 
 Drop DropDownMenu.swift, DropDownMenuCell.swift, DropDownTitleView.swift and DropDownMenuKit.xcassets into your Xcode project.
+
```

Comment:
Would be better to keep README untouched. 

On Github, there is a pane that lists all contributors at the top of each project, so many projects using Github don't maintain a separate list of contributors and I follow the same approach.